### ISSUE TITLE
Installation error while doing installation

### ISSUE THREAD

**Issue by sandeepl337** [AccessManager/access-manager#9]
Hello, 

Please look out the follow errror while executing installation process. 

Writing lock file
Generating autoload files
PHP Fatal error:  Cannot redeclare makeExpiry() (previously declared in /var/www/html/access-manager/laravel/vendor/access-manager/am-h
elpers/src/Helpers.php:23) in /var/www/html/access-manager/laravel/app/helpers.php on line 28
Script php laravel/artisan clear-compiled handling the post-install-cmd event returned with an error

  [RuntimeException]
  Error Output: PHP Fatal error:  Cannot redeclare makeExpiry() (previously declared in /var/www/html/access-manager/laravel/vendor/
  access-manager/am-helpers/src/Helpers.php:23) in /var/www/html/access-manager/laravel/app/helpers.php on line 28

create-project [-s|--stability="..."] [--prefer-source] [--prefer-dist] [--repository-url="..."] [--dev] [--no-dev] [--no-plugins] [--n
o-custom-installers] [--no-scripts] [--no-progress] [--keep-vcs] [--no-install] [--ignore-platform-reqs] [package] [directory] [version
]

**Comment by sandeepl337** [comment]
Here is my PHP version 

PHP 5.5.9-1ubuntu4.7 (cli) (built: Mar 16 2015 20:47:39)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.3, Copyright (c) 1999-2014, by Zend Technologies

**Comment by ayprograms** [comment]
i am also facing this same problem

**Comment by rahulmkhj** [comment]
fixed in 2.5.0

**Comment by ayprograms** [comment]
Hi Rahul, thanks for your reply, i fixed the problem that i ran into and my
installation is working now perfectly. I wish to join you in the
development of this project. what do you say?



*Ayuk Etta A. (B. Eng)*

*CTO, Piar inc*

*Softwer Engineer, ActivSpaces*


*Developer, OpenMRS*

**Comment by rahulmkhj** [comment]
@ayprograms You're welcome if you want to contribute to the project. How are you willing to contribute?

**Comment by ayprograms** [comment]
Hi Rahul,

i am thinking about contributing in the code direction. We have started a
wireless project here in Cameroon using Mikrotic and Access Manager seems
to solve some of our problems, however, i have evaluated the project and i
think alot can be added to it to solve other issues. i would loved that you
add me to your development team, send me your initial conception of the
project, your projection of the project. i think we can start off from
there. i have forked the repository on GitHub and i have the local code, i
am going through right now. I have also installed the version 2.4 on my
ubuntu server with radius manager which i am testing on my hotspot. Working
with you, i think we can accomplish more.

Regards



*Ayuk Etta A. (B. Eng)*

*CTO, Piar inc*

*Softwer Engineer, ActivSpaces*


*Developer, OpenMRS*
